geometry_msgs/msg/PolygonInstance.msg

```diff
@@ -0,0 +1,5 @@
+# A specification of a polygon where the first and last points are assumed to be connected
+# Includes and unique identification field for disambiguating multiple instances
+
+Point32[] points
```

Comment:
Looking at this in Waffle, we think that this should be a `Polygon`; that way, APIs that take a `geometry_msgs::msg::Polygon` can still work with this.